common/lib/capa/capa/templates/drag_and_drop_input.html

```diff
@@ -17,14 +17,14 @@
     <input type="text" name="input_${id}" id="input_${id}" aria-describedby="answer_${id}" value="${value|h}"
     style="display:none;"/>
 
-    <p class="status" aria-describedby="input_${id}">
-    ${status.display_name}
+    <p class="status drag-and-drop--status" aria-describedby="input_${id}">
+        <span class="sr">${status.display_name}</span>
     </p>
 
     <p id="answer_${id}" class="answer"></p>
 
     % if msg:
-        <span class="message">${HTML(msg)}</span>
+        <span class="message" aria-describedby="input_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
This is the old Drag and Drop problem, which is unsupported and being replaced by the new Drag and Drop. I don't think we need to worry about the aria-describedby attributes in this template.

FYI, it looks like Brian Talbot added in the aria-describedy attribute on the "status" (line 20) back in 2013. I don't think it makes sense though, as the input is a hidden field with no actual data. @cptvitamin, am I missing something here?